src/Entity.cpp

```diff
@@ -366,6 +344,28 @@ bool Entity::takeHit(Hazard &h) {
 	}
 
 	// misses cause reduced damage
+	int accuracy = h.accuracy;
+	if(powers->powers[h.power_index].mod_accuracy_mode == STAT_MODIFIER_MODE_MULTIPLY)
+		accuracy *= powers->powers[h.power_index].mod_accuracy_value / 100;
```

Comment:
I don't think you can use *= here. With it, this line breaks down to:

```
accuracy = accuracy * (powers->powers[h.power_index].mod_accuracy_value / 100);
```

When it should be:

```
accuracy = (accuracy * powers->powers[h.power_index].mod_accuracy_value) / 100;
```

These are not the same because of integer division. It worked in the latter way before because multiplication and division precedence is equal and done left-to-right. So I would add those parenthesis in there to clarify that.